mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -16,6 +16,8 @@ impl BridgeController {
     }
 
     pub async fn start(&mut self) -> Result<()> {
+        info!("starting bridge");
```

Comment:
I feel it is a better pattern to have the "starting xxx" logging inside the actual start method